deployments/helm/cvmfs-csi/values.yaml

```diff
@@ -1,107 +1,254 @@
-      #     - cvmfs_preload -u http://cvmfs-stratum-zero.cern.ch/cvmfs/repository.cern.ch -r /mnt/cvmfs/aliencache -k /etc/cvmfs/keys/cern.ch
-
-# Advanced settings
-# Changing these settings is not advised unless you know what you are doing
-csiPlugin:
-  # This image includes the patch to the plugin allowing
-  # for multiple PVCs across multiple namespaces
-  # https://github.com/cernops/cvmfs-csi/commit/c680862082738aba1469707a891a6f7fecf72f53
-  image: cloudve/csi-cvmfsplugin:v1.0.1
-  args:
-    - "--nodeid=$(NODE_ID)"
-    - "--endpoint=$(CSI_ENDPOINT)"
-    - "--v=5"
-    - "--drivername=cvmfs.csi.cern.ch"
-  pluginDirectory: /var/lib/kubelet/plugins/cvmfs.csi.cern.ch
-  nodeDriverImage: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
-csiAttacher:
-  image: quay.io/k8scsi/csi-attacher:v1.0.1
-  args:
-    - "--v=5"
-    - "--csi-address=$(ADDRESS)"
-csiProvisioner:
-  image: quay.io/k8scsi/csi-provisioner:v1.0.1
-  args:
-    - "--provisioner=cvmfs.csi.cern.ch"
-    - "--csi-address=$(ADDRESS)"
-    - "--v=5"
+    volumeSpec:
+      persistentVolumeClaim:
+        claimName: cvmfs-alien-cache
+
+# CSI Node plugin DaemonSet configuration.
+# Node plugin handles node-local operations, e.g. mounting and unmounting
+# CVMFS repositories.
+nodeplugin:
+
+  # Component name. Used as `component` label value
+  # and to generate DaemonSet name.
+  name: nodeplugin
+
+  # CVMFS CSI image and container resources specs.
+  plugin:
+    image:
+      repository: registry.cern.ch/magnum/cvmfs-csi
+      tag: v2.0.0
+      pullPolicy: IfNotPresent
+    resources: {}
+
+  # csi-node-driver-registrar image and container resources specs.
+  registrar:
+    image:
+      repository: registry.k8s.io/sig-storage/csi-node-driver-registrar
+      tag: v2.5.1
+      pullPolicy: IfNotPresent
+    resources: {}
+
+  # DaemonSet update strategy.
+  updateStrategySpec:
+    # When CVMFS Node plugin Pod is restarted, all existing CVMFS mounts on
+    # that node will break. If the Node plugin DaemonSet needs to be updated,
+    # all Pods that mount CVMFS volumes on that node must be restarted (deleted)
+    # too in order to refresh the mounts.
+    type: OnDelete
+
+  # Pod priority class name.
+  priorityClassName: system-node-critical
+
+  # Pod node selector.
+  nodeSelector: {}
+
+  # Pod node tolerations.
+  tolerations: []
+
+  # Pod node affinity.
+  affinity: {}
+
+  # hostPID must be enabled for automounts to work. If disabled,
+  # it is only possible to do single-repository mounts using
+  # `repository` volume attribute (see docs/how-to-use.md for
+  # more details).
+  hostPID: true
+
+  # (deprecated) ServiceAccount to use with Node plugin DaemonSet.
+  # New CVMFS CSI deployments do not need this. It is only necessary
+  # when upgrading from v1 of the driver.
+  serviceAccount:
+
+    # Name of the ServiceAccount (to use and/or create).
+    # If no name is provided, Helm chart will generate one.
+    serviceAccountName: cvmfs-nodeplugin
```

Comment:
This should be just `name` (instead of `serviceAccountName`, because the template uses this:
`.Values.nodeplugin.serviceAccount.name`

Reply:
The comment should also be updated, because the Helm chart does not "generate" a value.

Reply:
Thanks @jacksgt for noticing! There's named template `cvmfs-csi.serviceAccountName.nodeplugin` in _helpers.tpl that outputs the service account name -- I forgot to use it.